loader.js

```diff
@@ -0,0 +1,56 @@
+var carContainer;
+var counter = null
+var userInput = document.querySelector('input')
+var CarLot = (function () {
+  var inventory = [];
+  return {
+    getInventory: function () {
+      return inventory
+    },
+    loadInventory: function (callbackFunction) {
+      var inventoryLoader = new XMLHttpRequest();
+      inventoryLoader.open('GET', "inventory.json");
+      inventoryLoader.send();
+      inventoryLoader.addEventListener("load", function () {
+      inventory = JSON.parse(this.responseText).cars;
+      callbackFunction(inventory)
+      });
+    }
+  };
+  return CarLot
+})();
+
+CarLot.loadInventory(populateLot);
+
+function populateLot(){
+  var row = document.querySelector('.row')
+  inventory = CarLot.getInventory();
```

Comment:
When you pass `populateLot` into `loadInventory` as a callback, you're calling it and passing in `inventory`, but then in the definition of `populateLot` you haven't written it to accept an argument. instead you call `getInventory()`. It works, but your syntax indicates you're not sure why it works.